Riot/Modules/MatrixKit/Models/Room/MXKRoomDataSource.m

```diff
@@ -2885,6 +2885,11 @@ - (BOOL)canPerformActionOnEvent:(MXEvent*)event
         return YES;
     }
     
+    // Specific case for voice broadcast event
+    if (event.eventType == MXEventTypeCustom && [event.type isEqualToString:VoiceBroadcastSettings.voiceBroadcastInfoContentKeyType]) {
```

Comment:
There are two visible tiles for broadcasts: For the started and the ended state event. I think reactions should only work on the started event. We should add a check for it here.

Reply:
Why should we treat them any different?

Reply:
The end state event is only a notification in the timeline. I would say there should not be any action available for it. The start event is the visible tile. Here, reactions make sense. You can find out the type by checking the `state` key in the event content ([see discussion](https://github.com/vector-im/element-meta/discussions/632)).

There are also other events (with states resumed and paused)  we should avoid enabling reactions for, event if they should not be visible. I don't know what happens if you enable to show hidden messages and then react on them.

Can only provide screenshots from web. But I hope you see what I mean:

Voice Broadcast Info event `started` state:

![image](https://user-images.githubusercontent.com/6216686/211051027-458e9af9-1f88-4fd4-8749-318b61c2c059.png)

Voice Broadcast Info event `stopped` state:

![image](https://user-images.githubusercontent.com/6216686/211051063-95e6847e-2a93-4923-b382-6ce7a6eea187.png)

Reply:
Sounds good to me, mind making this change @nimau